### PR TITLE
Update database schema for efficient jobs deletion

### PR DESCRIPTION
**Pull request by lpellegr**

### CODE REVIEW

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
@ow2-proactive/developers Could two people please review the changes?

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java [hunk 1]
```diff
@@ -6,21 +6,7 @@
...
-import javax.persistence.FetchType;
-import javax.persistence.Index;
-import javax.persistence.JoinColumn;
-import javax.persistence.Lob;
-import javax.persistence.ManyToOne;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import javax.persistence.Table;
-import javax.persistence.Transient;
+import javax.persistence.*;
```
Comment (tobwiens) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java]: You can setup your idea to not .* import.

**Comment by tobwiens** [comment]
OK :+1:

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java (hunk 1 above)
Comment (lpellegr) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java]: I use our [coding-rules configuration](https://github.com/ow2-proactive/coding-rules/blob/master/settings.jar) that uses the wildcard if there are more than 15 imports for the same package. I agree that wildcard may lead to confusion. We should keep that in mind for our new coding rules. I think Google profile don't use/allow wildcard by default.  
capture d ecran de 2016-05-09 11-59-04

**Comment by mbenguig** [comment]
In the updated code sections, is there any "unique value field" that is currently not indexed ?...

**Comment by mbenguig** [comment]
Ok for me.

**Comment by lpellegr** [comment]
@mbenguig No, unique field values should be indexed by default whatever the database engine is.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by tobwiens** [comment]
@lpellegr I see new commits (after the review ok comments) Do you like another review?

**Comment by lpellegr** [comment]
@tobwiens There are new commits after the review because I have rebased with master. It should be ok, thank you.

**Pull request merged**